.appveyor.yml

```diff
@@ -33,18 +34,14 @@ environment:
     - WINPYTHON: "Python38"
       COVERAGE: 0
 
+    - WINPYTHON: "Python310"
+      COVERAGE: 0
+
+
 # remove sets of build jobs based on criteria below
 # to fine tune the number and platforms tested
 matrix:
   exclude:
-    # test python 3.5 on Visual Studio 2015 image
-    - image: Visual Studio 2015
-      WINPYTHON: "Python36"
-    - image: Visual Studio 2015
-      WINPYTHON: "Python37"
-    - image: Visual Studio 2015
-      WINPYTHON: "Python38"
-
     # test python 3.8 on Visual Studio 2017 image
     - image: Visual Studio 2017
```

Comment:
python 310?

Reply:
2nd try, let's see if I've got it right this time.